### ISSUE TITLE
New Version Install Issues

### ISSUE THREAD

**Issue by KitchM** [giuspen/cherrytree#1315]
**Version, Operative system**
Debian 10 (latest update), Xfce 4.12, on AMD64

**Describe the bug**
Deb file install cannot complete because of unmet dependencies.

**To Reproduce**
1. Go to terminal window
2. Enter sudo apt install ./cherrytree_0.99.18-2_amd64.deb
3. See error

sudo apt install ./cherrytree_0.99.18-2_amd64.deb
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Note, selecting 'cherrytree' instead of './cherrytree_0.99.18-2_amd64.deb'
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 cherrytree : Depends: libgspell-1-2 (>= 1.8.2) but it is not installable
              Depends: libc6 (>= 2.30) but 2.28-10 is to be installed
              Depends: libgcc-s1 (>= 3.0) but it is not installable
              Depends: libglibmm-2.4-1v5 (>= 2.64.2) but 2.58.0-2 is to be installed
              Depends: libstdc++6 (>= 9) but 8.3.0-6 is to be installed

[By the way, there is no current maintainer for Cherrytree on Debian package repos.]

**Comment by ciranus** [comment]
You're right. I did not notice this earlier in Debian because using Bullseye (testing).

> For distributions older than Ubuntu 20.04 the **flatpak** will be available soon
> https://github.com/giuspen/cherrytree/issues/1271#issuecomment-713090914

> there is no current maintainer for Cherrytree on Debian package repos

It's coming.....
https://salsa.debian.org/debian/cherrytree

If you are familiar with Debian, I think with limited risk, you can try this:
add to /etc/apt/sources.list

`deb http://deb.debian.org/debian/ bullseye main`
then
```
# apt update
# apt install ./cherrytree_0.99.18-2_amd64.deb
```
And then delete or comment this line in sources.list
`# deb http://deb.debian.org/debian/ bullseye main`

**Comment by merkys** [comment]
This is not an issue of CherryTree. It seems that you have downloaded `cherrytree_0.99.18-2_amd64.deb`, which is prepared for Debian unstable, thus it uses dependencies that are not available on Debian 10.

> [By the way, there is no current maintainer for Cherrytree on Debian package repos.]

I have recently taken over the maintenance for CherryTree on Debian. I will try to look into backporting it also to Debian 10.

**Comment by KitchM** [comment]
Thank you everyone.  I will look forward to the backport.

**Comment by merkys** [comment]
@KitchM, could you maybe close the issue then?

**Comment by ciranus** [comment]
> I will look forward to the backport.

Debian is currently doing the job.
https://salsa.debian.org/debian/cherrytree

**Comment by KitchM** [comment]
@ciranus  I believe you mean that merkys is doing the job.

@merkys  the issue is not fixed until the backport is done.  No need to close this issue yet.  Please keep me posted.

**Comment by ciranus** [comment]
> @ciranus I believe you mean that merkys is doing the job.

Ha yes sorry !!!!
